python/lsst/sims/photUtils/Photometry.py

```diff
@@ -15,458 +15,110 @@
 from collections import OrderedDict
 import lsst.utils
 from lsst.sims.photUtils import Sed, Bandpass, LSSTdefaults, calcGamma, \
-                                calcMagError_m5, PhotometricParameters, magErrorFromSNR
+                                calcMagError_m5, PhotometricParameters, \
+                                magErrorFromSNR, BandpassDict
 from lsst.sims.utils import defaultSpecMap
 
-__all__ = ["PhotometryHardware", "PhotometryBase"]
+__all__ = ["loadBandpassesFromFiles", "loadTotalBandpassesFromFiles"]
```

Comment:
We could leave the functions here and still have the classmethods we discussed. That would just call these functions. That way, if you want to use these functions separately, you could.